plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -342,7 +342,7 @@ func ClusterRoles() []rbac.ClusterRole {
 				// this is for leaderlease access
 				// TODO: scope this to the kube-system namespace
 				rbac.NewRule("create").Groups(legacyGroup).Resources("endpoints").RuleOrDie(),
-				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-scheduler").RuleOrDie(),
+				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-scheduler-default-scheduler").RuleOrDie(),
```

Comment:
the name is redundant.  

Why wouldn't we change the pattern to: "kube-s.SchedulerName-scheduler"  - 
"kube-default-scheduler"